src/i18n/Resources/Shared.en.resx

```diff
@@ -117,4 +117,576 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Phone" xml:space="preserve">
+    <value>Phone</value>
+    <comment>Label prompting a customer to enter their telephone number</comment>
+  </data>
+  <data name="Requested date" xml:space="preserve">
+    <value>Requested date</value>
+    <comment>Label prompting a customer to select a date</comment>
+  </data>
+  <data name="Requested date and time" xml:space="preserve">
+    <value>Requested date and time</value>
+    <comment>Label to show a customer what date and time they selected</comment>
+  </data>
+  <data name="Requested time" xml:space="preserve">
+    <value>Requested time</value>
+    <comment>Label prompting a customer to select a time</comment>
+  </data>
+  <data name="Subject" xml:space="preserve">
+    <value>Subject</value>
+    <comment>Label next to a drop-down with a list of subjects</comment>
+  </data>
+  <data name="Back" xml:space="preserve">
+    <value>Back</value>
+    <comment>Navigation button: go back to a previous page</comment>
+  </data>
+  <data name="Back to the site" xml:space="preserve">
+    <value>Back to the site</value>
+    <comment>Navigation button: go back to the Web site</comment>
+  </data>
+  <data name="Go" xml:space="preserve">
+    <value>Go</value>
+    <comment>Navigation button: perform a search or other action</comment>
+  </data>
+  <data name="Next" xml:space="preserve">
+    <value>Next</value>
+    <comment>Navigation button: go forward to the next page in a process</comment>
+  </data>
+  <data name="Submit" xml:space="preserve">
+    <value>Submit</value>
+    <comment>Button: submit form data</comment>
+  </data>
+  <data name="Subscribe" xml:space="preserve">
+    <value>Subscribe</value>
+    <comment>Button: subscribe to a mailing list</comment>
+  </data>
+  <data name="Use my location" xml:space="preserve">
+    <value>Use my location</value>
+    <comment>Button: engage a mobile device's GPS to look up the customer's location</comment>
+  </data>
+  <data name="Connect with us on social media!" xml:space="preserve">
+    <value>Connect with us on social media!</value>
+    <comment>Call to action: visit social media accounts</comment>
+  </data>
+  <data name="Connect with us on {0}" xml:space="preserve">
+    <value>Connect with us on {0}</value>
```

Comment:
This text causes the facebook and twitter icons to be hidden by my adblock (Adblock Plus on Firefox). Specifically it adds css to hide the display of elements with the title "Connect with us on Facebook" (or Twitter). Not sure if it's an actual issue to make changes for, figured it was worth mentioning.